src/panels/filter-panel/value_select_dropdown.js

```diff
@@ -0,0 +1,314 @@
+import angular from 'angular';
+import _ from 'lodash';
+
+import coreModule from 'app/core/core_module';
+
+export class OnmsValueSelectDropdownCtrl {
+  /*
+  dropdownVisible: any;
```

Comment:
Is this for typescript?

Reply:
Yes and no, it's because I ported the grafana internal `value_select_dropdown.ts` to pure javascript because I don't have access to the objects directly.  I left everything as in-place as possible and just commented some stuff out.